src/v3/src/transformer/selectAuthenticator/transformSelectAuthenticatorVerify.ts

```diff
@@ -75,7 +75,6 @@ export const transformSelectAuthenticatorVerify: IdxStepTransformer = ({
   const informationalTextElement: DescriptionElement = {
     type: 'Description',
     contentType: 'subtitle',
-    noMargin: true,
```

Comment:
This was done to fix the lack of bottom margin that Alex was talking about, right?